### PR TITLE
Spring boot tracing filter

### LINKED ISSUE #1965: elasticjob.tracing.type=RDB does not support configuration specific jobs?
when i use module "elasticjob-lite-spring-boot-starter" and configure elasticjob.tracing.type=RDB  does not support configuration specific jobs? Currently, all global jobs are effective

### PR DESCRIPTION
**Pull request by sunkai-cai**
Fixes #1965.

Changes proposed in this pull request:
-  adding a black-and-white list in the `elasticjob-lite-spring-boot-starter` confing

### CODE REVIEW

File: elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/tracing/TracingProperties.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Getter
+@Setter
+@ConfigurationProperties(prefix = "elasticjob.tracing")
+public class TracingProperties {
+    
+    private String type;
+    
+    private String[] includeJobNames;
```
Comment (TeslaCN) [inline elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/tracing/TracingProperties.java]: Could we consider using Set here?

**Review by TeslaCN — CHANGES_REQUESTED** [review]

File: elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/tracing/TracingProperties.java (hunk 1 above)
Comment (sunkai-cai) [reply elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/tracing/TracingProperties.java]: good idea

File: elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/job/ElasticJobBootstrapConfiguration.java [hunk 2]
```diff
@@ -95,10 +95,25 @@ private void constructJobBootstraps(final ElasticJobProperties elasticJobPropert
         }
     }
     
+    private void jobExtraConfigurations(final JobConfiguration jobConfig, final TracingConfiguration<?> tracingConfig) {
+        if (null == tracingConfig) {
+            return;
+        }
+        TracingProperties tracingProperties = applicationContext.getBean(TracingProperties.class);
```
Comment (TeslaCN) [inline elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/job/ElasticJobBootstrapConfiguration.java:102]: What if user configured both `include` and `exclude`? I prefer throwing an exception.

**Review by TeslaCN — CHANGES_REQUESTED** [review]

File: elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/job/ElasticJobBootstrapConfiguration.java (hunk 2 above)
Comment (sunkai-cai) [reply elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/job/ElasticJobBootstrapConfiguration.java:102]: you are right

File: elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/job/ElasticJobBootstrapConfiguration.java [hunk 3]
```diff
@@ -95,10 +95,30 @@ private void constructJobBootstraps(final ElasticJobProperties elasticJobPropert
...
+        if (null == tracingProperties.getIncludeJobNames() && null == tracingProperties.getExcludeJobNames()) {
+            jobConfig.getExtraConfigurations().add(tracingConfig);
+        } else if (null != tracingProperties.getIncludeJobNames()) {
+            if (tracingProperties.getIncludeJobNames().contains(jobConfig.getJobName())) {
+                jobConfig.getExtraConfigurations().add(tracingConfig);
+            }
+        } else if (null != tracingProperties.getExcludeJobNames()
+                && !tracingProperties.getExcludeJobNames().contains(jobConfig.getJobName())) {
+            jobConfig.getExtraConfigurations().add(tracingConfig);
+        }
```
Comment (TeslaCN) [inline elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/job/ElasticJobBootstrapConfiguration.java]: I think these logic can be more elegant.

Comment (sunkai-cai) [reply elasticjob-lite/elasticjob-lite-spring/elasticjob-lite-spring-boot-starter/src/main/java/org/apache/shardingsphere/elasticjob/lite/spring/boot/job/ElasticJobBootstrapConfiguration.java]: Ok, I try it.

**Review by TeslaCN — APPROVED** [review]

**Pull request merged**